escrow/src/interface.sw

```diff
@@ -0,0 +1,161 @@
+library interface;
+
+dep data_structures;
+
+use data_structures::{Arbiter, Asset};
+use std::{identity::Identity, vec::Vec};
+
+abi Escrow {
+    /// Buyer accepts proposal to change arbiter details
+    ///
+    /// # Arguments
+    ///
+    /// * `identifier` - Identifier used to find a specific escrow
+    ///
+    /// # Reverts
+    ///
+    /// * When the escrow is not in the State::Pending state
+    /// * When the caller is not the buyer
+    /// * When the arbiter has not been proposed by the seller
+    #[storage(read, write)]fn accept_arbiter(identifier: u64);
```

Comment:
I'm unsure what you mean by the last comment nevertheless once a script factory is a thing then identifiers will be unneccessary so that will be removed.